dom.bs

```diff
@@ -5746,6 +5777,9 @@ null if <var>event</var>'s <a>composed flag</a> is unset and <a for=/>shadow roo
 <dfn for=ShadowRoot export><code>onslotchange</code></dfn> <a>event handler</a>, whose
 <a>event handler event type</a> is {{HTMLSlotElement/slotchange}}.
 
+<p>The <dfn attribute for=ShadowRoot><code>slotAssignment</code></dfn> attribute's getter must
```

Comment:
Nit I can fix: The X getter steps are to return*

(Once this lands I can take care of the other getters in this section. It seems slightly preferable to have this correct from the start for blame.)